Src/ILGPU.Tests/Configurations.txt

```diff
@@ -1,13 +1,14 @@
-﻿ArrayViews: Debug, Release, O2
```

Comment:
ISSUE: Looks like there was an issue here with the merge with the master branch.

You've added `ExchangeBufferOperations: Debug, Release` but at the same time, someone elese added `, O2` to all the other lines.

Reply:
Alright, I'll pull from the master branch to resolve these issues.